src/index.html

```diff
@@ -132,17 +133,62 @@ <h1 class="font-sans text-6xl font-extrabold text-fuchsia-950">
 
             <!-- Social media review cards beginning -->
 
-            <div class="grid grid-cols-3 gap-6">
-                <div></div>
+            <div class="flex justify-evenly card">
 
```

Comment:
Hello Benjamin,

Looking at the code below, you did not pull the latest version of the repository . Your card review section looks completely different from the original version. Kindly pull the latest one for the card review section. I am hoping we can just stick to using the Tailwind breakpoints for the entire file instead of the custom CSS. Thank you.

Reply:
Alright sure. That will mean you will have to build from scratch using the mobile first design approach but if not, I will be looking forward to you using tailwinds custom CSS breakpoints so I learn from you.